gui/directoryservice/forms.py

```diff
@@ -658,6 +662,12 @@ def save(self):
         else:
             if started is True:
                 started = notifier().stop("activedirectory")
+
+        if enable_monitoring and enable:
+            notifier().enable_service_monitoring(monit_frequency, monit_retry, fqdn, 3268, 'activedirectory')
+        else:
+            notifier().disable_service_monitoring(monit_frequency, monit_retry, fqdn, 3268, 'activedirectory')
```

Comment:
Any reason to not contact middlewared directly?

My goal was to slowly deprecate notifier completely, since thats one of the reasons middlewared was created.